src/org/omegat/Main.java

```diff
@@ -277,14 +281,26 @@ protected static int runGUI() {
             // do nothing
         }
 
+        String theme = Preferences.getPreferenceDefault(Preferences.THEME_SELECTED_NAME, "Default");
```

Comment:
I'm a bit wary of expecting `Preferences` to be initialized this early. But if it's working then I guess it's OK.

Reply:
Preferences is initialized in Main:172 so it's working.